tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -282,6 +271,16 @@ sub prepare_guest_for_sriov_passthrough {
         script_run "ssh root\@$vm \"sed -i '/udev_log *=/{h;s/^[# ]*udev_log *=.*\\\$/udev_log=debug/};\\\${x;/^\\\$/{s//udev_log=debug/;H};x}' $udev_conf_file\"";
     }
 
+    # Journals with prvious reboot
```

Comment:
minor typo: prvious -> previous

Reply:
Why not considering putting guest journald setup to lower level base or lib, so that otherwhere can have/use it too?

Reply:
corrected. thanks.